tools/single_use/Cleanup_multiple_firstVisits.php

```diff
@@ -27,7 +27,7 @@
                   WHERE VisitNo=1 
                   GROUP BY CandID 
                   HAVING COUNT(CandID) > 1";
-$candIDs      = $db->pselect($query_candID, array());
+$candIDs      = $db->pselect($query_candID, []);
```

Comment:
I'm not sure I understand the question. I think we can still modify single_use tools because someone could be updating from a very old version of LORIS. In any case this won't cause any operations in the code.